pkg/cmd/cmd.go

```diff
@@ -36,8 +36,8 @@ func NewCmd() *cobra.Command {
 func Run() {
 	cmd := NewCmd()
 
-	// Outputs cmd.Print to stdout.
-	cmd.SetOut(os.Stdout)
+	// Outputs cmd.Print to stderr.
+	cmd.SetOut(os.Stderr)
```

Comment:
Could you redrict err output to stderr?

```go
cmd.SetOut(os.Stdout)
cmd.SetErr(os.Stderr)
...
cmd.PrintErr(err) // line 48
```

Reply:
Yes, use `cmd.PrintErr` to print error message to stderr is more natural. And maybe we should use `cmd.PrintErrln` here to print error with new line.

Reply:
Yes, let's upgrade cobra.